tests/phpunit/Compat/WC_Stripe_Subscription_Initial_Test.php

```
@@ -0,0 +1,145 @@
+		// Assert: nothing was returned.
+		$this->assertEquals( 'success', $result['result'] );
+		$this->assertArrayHasKey( 'redirect', $result );
+
+		$order = wc_get_order( $order_id );
+
+		$actual = $order_helper->get_stripe_intent_id( $order );
+
+		// Clean up.
+		remove_filter( 'pre_http_request', $pre_http_request_response_callback, 10 );
+		$_POST = $old_post;
+		WC_Stripe_Helper::delete_main_stripe_settings();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\WC_Stripe_Helper' in method 'test_initial_intent_parameters'. (undefined)

(StaticAccess)